unit_tests/tests/actuators/test_boost.cpp

```diff
@@ -64,7 +64,7 @@ TEST(BoostControl, OpenLoop) {
 	bc.init(nullptr, &openMap, nullptr, nullptr);
 
 	// Should pass TPS value thru
-	Sensor::setMockValue(SensorType::DriverThrottleIntent, 47.0f);
+	Sensor::setMockValue(SensorType::Tps1, 47.0f);
```

Comment:
@rusefillc, could you please confirm that this change in boost control open loop behavior is expected?